inst/tinytest/testApp.R

```diff
@@ -91,9 +91,23 @@ expect_identical_xl(
 )
 
 expect_identical_xl(
-  vapply(studies[[1]][["plots"]][[1]][["plots"]],
-         function(x) x[["plotType"]], character(1)),
-  c("singleFeature", "multiFeature")
+  studies[[1]][["plots"]][[1]][["plots"]][[1]][["plotType"]],
+  "singleFeature"
+)
+
+expect_identical_xl(
+  studies[[1]][["plots"]][[1]][["plots"]][[2]][["plotType"]],
+  "multiFeature"
+)
+
+expect_identical_xl(
+  studies[[1]][["plots"]][[1]][["plots"]][[3]][["plotType"]],
+  "multiTest"
+)
+
+expect_identical_xl(
+  studies[[1]][["plots"]][[1]][["plots"]][[4]][["plotType"]],
+  list("multiFeature", "multiTest")
```

Comment:
I don't love that this gets returned as a list, but I don't see any other way around it because of how the JSON is converted. I guess it's ok since `listStudies()` is mainly used by the FE

Reply:
An alternative would be providing the test results outside the list item "results" from getPlottingData. As such, in the output of getPlottingData we could have list items like "features", "samples", "assays" and further list items each containing the results for a single test. However, in my opinion, this looks less organized in contrast to having all test results placed within a sublist object "results", which is currently implemented. Please let me know your thoughts here.

Reply:
We're talking about different things. I was referring to the output from `listStudies()`. I was "thinking out loud". I don't want to change the way you are returning data with `getPlottingData()`. Essentially we need to make sure Paul is expecting this on the FE, that's all.